src/providers/types.ts

```diff
@@ -1,68 +1,74 @@
+import { TariUniverseProvider } from "./tari_universe";
 
 export type SubstateRequirement = {
-    substate_id: string,
-    version?: number | null,
+  substate_id: string;
+  version?: number | null;
 };
 
 export type SubmitTransactionRequest = {
-    account_id: number,
-    instructions: object[],
-    fee_instructions: object[],
-    inputs: object[],
-    input_refs: object[],
-    required_substates: SubstateRequirement[],
-    is_dry_run: boolean,
-    min_epoch: number | null,
-    max_epoch: number | null,
+  account_id: number;
+  instructions: object[];
+  fee_instructions: object[];
+  inputs: object[];
+  input_refs: object[];
+  required_substates: SubstateRequirement[];
+  is_dry_run: boolean;
+  min_epoch: number | null;
+  max_epoch: number | null;
 };
 
 export type SubmitTransactionResponse = {
-    transaction_id: string;
+  transaction_id: string;
 };
 
-
 export type TransactionResult = {
-    transaction_id: string;
-    status: TransactionStatus;
-    result: object | null;
+  transaction_id: string;
+  status: TransactionStatus;
+  result: object | null;
 };
 
 export enum TransactionStatus {
-    New,
-    DryRun,
-    Pending,
-    Accepted,
-    Rejected,
-    InvalidTransaction,
-    OnlyFeeAccepted
+  New,
```

Comment:
more unnecessary changes to linter conventions - please revert this to original state